src/utils/utils.cpp

```diff
@@ -409,6 +603,57 @@ void EchoInfo(const std::string &message) {
   }
 }
 
+void EchoStats(const std::map<std::string, std::int64_t> &stats) {
+  for (const auto &[key, value] : stats) {
+    if (value == 0) {
+      continue;
+    } else if (key == "nodes-created") {
+      std::cout << value << " nodes have been created." << std::endl;
+    } else if (key == "nodes-deleted") {
+      std::cout << value << " nodes have been deleted." << std::endl;
+    } else if (key == "relationships-created") {
+      std::cout << value << " relationships have been created." << std::endl;
+    } else if (key == "relationships-deleted") {
+      std::cout << value << " relationships have been deleted." << std::endl;
+    } else if (key == "labels-added") {
+      std::cout << value << " labels have been created." << std::endl;
+    } else if (key == "labels-removed") {
+      std::cout << value << " labels have been deleted." << std::endl;
+    } else if (key == "properties-set") {
+      std::cout << value << " properties have been updated." << std::endl;
+    }
+  }
+}
+
+void EchoNotification(const std::map<std::string, std::string> &notification) {
+  if (notification.at("severity") == "WARNING") {
```

Comment:
you can still store this into a `std::string_view` so you don't have to fetch it every time.